.github/workflows/apps/appsec-test-contrib-submodules.arm64.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+# This script is used to test the contrib submodules in the apps directory.
+# It is run by the GitHub Actions CI workflow defined in
+# .github/workflows/appsec.yml.
+
+echo "Running appsec ARM64 tests for:"
+echo "  V2_BRANCH=$V2_BRANCH"
+echo "  CGO_ENABLED=$CGO_ENABLED"
+echo "  DD_APPSEC_ENABLED=$DD_APPSEC_ENABLED"
+echo "  DD_APPSEC_WAF_TIMEOUT=$DD_APPSEC_WAF_TIMEOUT"
+
+docker run --platform=linux/arm64 -v $PWD:$PWD -w $PWD -eCGO_ENABLED=$CGO_ENABLED -eDD_APPSEC_ENABLED=$DD_APPSEC_ENABLED -eDD_APPSEC_WAF_TIMEOUT=$DD_APPSEC_WAF_TIMEOUT golang go test -v ./appsec/... ./internal/appsec/...
+
+SCOPES=("gin-gonic/gin" "google.golang.org/grpc" "net/http" "gorilla/mux" "go-chi/chi" "go-chi/chi.v5" "labstack/echo.v4")
+for SCOPE in "${SCOPES[@]}"; do
+  if [[ "$V2_BRANCH" == "true" ]]; then
+    cd "./v2/contrib/$SCOPE"
```

Comment:
Note for later when we all switch to v2: I wonder if we could get rid of this v2 directory just for contribs.

Reply:
@Julio-Guerra Unfortunately, it's a limitation of Go module specification. Any nested module must be found in a subdirectory that matches the part of the module's path after the repository root path. Source: [Go Modules Reference](https://go.dev/ref/mod#vcs-dir).

This means that for releasing nested modules in a v2 repository like [github.com/DataDog/dd-trace-go/v2](http://github.com/DataDog/dd-trace-go/v2) they must be in a path starting with v2/ like v2/contrib/gin-gonic/gin for [github.com/DataDog/dd-trace-go/v2/contrib/gin-gonic/gin](http://github.com/DataDog/dd-trace-go/v2/contrib/gin-gonic/gin).

This doesn't affect anything else that isn't a nested module.

The alternatives are:

- To create a new repository dd-trace-go-v2 like AWS did with [aws-sdk-go-v2](https://github.com/aws/aws-sdk-go-v2).
- Or to remove the go.mod at the repository’s root, to create a new in each product’s directory: appsec, datastreams, ddtrace, and profiler.

Reply:
Ok, I thought tagging with `v2/contrib/foo` was enough to give the path to follow in the repo :-(